### PR TITLE
Cloud + SWCC documentation

### PR DESCRIPTION
**Pull request by JakeWags**

### CODE REVIEW

**Review by akenmorris — APPROVED** [review]
Looks good to me.

**Comment by akenmorris** [comment]
@sheryjoe , would you like to review before we merge it?

**Comment by akenmorris** [comment]
Comments from @sheryjoe :
- I would use ShapeWorks-Cloud (with a hyphen)
- Login screen: we should add a link to cloud page.
- we should also add the walk-through video in the getting started page

**Comment by JakeWags** [comment]
@iyerkrithika21 @akenmorris I just added Group Differences to this PR. That feature is waiting for review in ShapeWorks-Cloud (PR: [SW-Cloud #300](https://github.com/girder/shapeworks-cloud/pull/300).

Before merging here we should merge the group-differences PR in ShapeWorks-Cloud.

File: docs/cloud/getting-started-with-cloud.md
Comment (iyerkrithika21) [inline docs/cloud/getting-started-with-cloud.md:1]: Could we add the link to the shapeworks cloud client in the login screen section?

File: docs/cloud/cloud-dataset.md
Comment (iyerkrithika21) [inline docs/cloud/cloud-dataset.md:1]: It would be nice to explain what the datasets contain: meshes and segmentations and the supported formats by the cloud platform.

File: docs/cloud/cloud-data-module.md
Comment (iyerkrithika21) [inline docs/cloud/cloud-data-module.md:1]: I am unclear on the classification of subjects vs. anatomies. Is this patient vs. multiple domains in the context of Shapeworks' definition?

File: docs/cloud/cloud-groom-module.md
Comment (iyerkrithika21) [inline docs/cloud/cloud-groom-module.md:1]: Do we want to link this to the documentation we have for the commands? 
URL: http://sciinstitute.github.io/ShapeWorks/latest/workflow/groom.html

**Review by iyerkrithika21 — APPROVED** [review]
Just a few minor comments. Overall the PR looks good!

File: docs/cloud/cloud-data-module.md
Comment (JakeWags) [reply docs/cloud/cloud-data-module.md:1]: I believe this is the case. Each subject represents one "scan" aka one patient. Multiple anatomies would be equivalent to having multiple domains.

**Comment by akenmorris** [comment]
In order to get this into the 6.4 docs we'll need to cherry pick the doc changes onto the 6.4 release branch. I tried to change the base, but it will have to bring in all of the post-6.4 refactoring.

**Comment by akenmorris** [comment]
I've rebased this into the release_v6.4.0 branch

**Pull request closed without merging**
